openapi/openapi.yaml

```diff
@@ -11,39 +11,39 @@ info:
     url: https://www.apache.org/licenses/LICENSE-2.0.html
   version: 0.0.1
 tags:
-- name: Bridges
-  description: "The API that allow the user to retrieve, create or delete Bridge instances."
-- name: Error Catalog
-  description: List and get the error definitions from the error catalog.
-- name: Processors
-  description: "The API that allow the user to retrieve, create or delete Processors\
+  - name: Bridges
```

Comment:
I think you will have to revert the changes in this file, otherwise the pipeline will fail